src/_tutorials/index.md

```diff
@@ -14,19 +14,14 @@ The following tours assume a basic familiarity with the Dart language,
 which you can get from skimming the
 [language samples](/samples) or the
 [language tour](/guides/language/language-tour).
+Next, learn about futures by following the
+asynchrony codelab: [Asynchrony: futures, async, await](/codelabs/async-await).
```

Comment:
One suggestion for reducing duplication of "asynchrony" here might be: 

Next, learn about futures in the following codelab: [Asynchrony: futures, async, await](/codelabs/async-await)